sass/includes/_maintenance-503.scss

```diff
@@ -17,7 +17,7 @@
         margin: 3rem 5rem;
     }
 
-    @media only screen and (max-width: 37.5rem) {
+    @media only screen and (width <= 37.5rem) {
```

Comment:
@ahosgood these are the media query changes that the automated fix added as discussed yesterday

Reply:
It's a really weird one. This is not the standard syntax so I have no idea why this change is being made. It's not part of Prettier, is it? I don't seem to get the same change on the `tna-frontend` repository...

Reply:
I'll investigate this one again.